.github/workflows/dbt_slim_ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         run: "dbt compile $PROFILE $COMPILE_TAG"
 
       - name: dbt seed
-        run: "dbt seed $PROFILE --select state:modified$TAG --exclude $EXCLUDE tag:prod_exclude --state ."
+        run: "dbt seed $PROFILE --select +state:modified${TAG} --exclude $EXCLUDE tag:prod_exclude --state ."
```

Comment:
you'll probably need `@` instead of `+` here to achieve what you're trying to do.
(seeds are not a parent from a model, but they are often parents from a test which is a child of a model..) 
We've done this before but removed it as it would bloat all PRs with almost all seeds.

Reply:
It's to fix this CI failure: https://github.com/duneanalytics/spellbook/actions/runs/5485534751/jobs/9994453915
Do you know what is up here?

Reply:
yeah the seeds are currently not materialized in prod.. :(
but I've not seen the previous problems we had with running those, so might be that we can just re-enable that schedule now that we're on dbt 1.4 instead of 1.2 :thinking: